lib/spack/spack/util/gpg.py

```diff
@@ -23,6 +23,16 @@
 GNUPGHOME = None
 
 
+class GPGKey:
+    def __init__(self, id, fingerprint="", uid=""):
```

Comment:
Should we do some validation of id, fingerprint, and uid here? I don't know how stable the gpg output is across versions, so in case it'll ever break, it might not hurt to do some basic checks whether id/fingerprint/uid are non-empty etc.